predictive-analytics/Getting-Started-with-R.md

```diff
@@ -606,14 +606,14 @@ I encourage you to explore the help information for RColorBrewer by entering hel
 
 This produces the following figure. The overlapping labels are due to the landscape aspect ratio.
 
-![All RColorBrewer Palettes](Getting-Started-with-R_images/img8.png)
+![All RColorBrewer Palettes in Predictive Analytics Getting Started](getting-started-with-r_images/predictive-analytics-getting-Started-with-r-all-rcolorbrewer-palettes.png)
```

Comment:
reduce image and alt text length in all places

Reply:
I have reduced image and alt text length in all places